pkg/webhook/webhook.go

```diff
@@ -157,49 +164,80 @@ func (w *Config) Start(ctx context.Context) error {
 
 // createFleetWebhookConfiguration creates the ValidatingWebhookConfiguration object for the webhook.
 func (w *Config) createFleetWebhookConfiguration(ctx context.Context) error {
-	whCfg := admv1.ValidatingWebhookConfiguration{
+	vWhCfg := admv1.ValidatingWebhookConfiguration{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: FleetWebhookCfgName,
+			Name: fleetValidatingWebhookCfgName,
 			Labels: map[string]string{
 				"admissions.enforcer/disabled": "true",
 			},
 		},
-		Webhooks: w.buildValidatingWebHooks(),
+		Webhooks: w.buildFleetValidatingWebhooks(),
 	}
 
 	// We need to ensure this webhook configuration is garbage collected if Fleet is uninstalled from the cluster.
 	// Since the fleet-system namespace is a prerequisite for core Fleet components, we bind to this namespace.
-	if err := bindWebhookConfigToFleetSystem(ctx, w.mgr.GetClient(), &whCfg); err != nil {
+	if err := bindWebhookConfigToFleetSystem(ctx, w.mgr.GetClient(), &vWhCfg); err != nil {
 		return err
 	}
 
-	if err := w.mgr.GetClient().Create(ctx, &whCfg); err != nil {
+	if err := w.mgr.GetClient().Create(ctx, &vWhCfg); err != nil {
 		if !apierrors.IsAlreadyExists(err) {
 			return err
 		}
-		klog.V(2).InfoS("validatingwebhookconfiguration exists, need to update", "name", FleetWebhookCfgName)
+		klog.V(2).InfoS("validating webhook configuration exists, need to update", "name", fleetValidatingWebhookCfgName)
 		// Here we simply use delete/create pattern to implement full overwrite
-		err := w.mgr.GetClient().Delete(ctx, &whCfg)
+		err := w.mgr.GetClient().Delete(ctx, &vWhCfg)
 		if err != nil {
 			return err
 		}
-		err = w.mgr.GetClient().Create(ctx, &whCfg)
+		err = w.mgr.GetClient().Create(ctx, &vWhCfg)
 		if err != nil {
 			return err
 		}
 		return nil
 	}
-	klog.V(2).InfoS("successfully created validatingwebhookconfiguration", "name", FleetWebhookCfgName)
+	klog.V(2).InfoS("successfully created validating webhook configuration", "name", fleetValidatingWebhookCfgName)
```

Comment:
please refactor these code out as a separate func, so that you don't need to duplicate the code for guard rail.